File: app/src/main/java/htw/gma_sose22/metronomprokit/beat/BeatPattern.java
package htw.gma_sose22.metronomprokit.beat;

import java.util.Date;
import com.fasterxml.jackson.annotation.JsonProperty;

public class BeatPattern {

    @JsonProperty("patternName")
    String patternName;

    @JsonProperty("createdAt")
    Date createdAt;

}

File: app/src/main/java/htw/gma_sose22/metronomprokit/beat/BeatPattern.kt
package htw.gma_sose22.metronomprokit.beat

import com.fasterxml.jackson.annotation.JsonProperty
import java.util.*

class BeatPattern {
    @JsonProperty("patternName")
    var patternName: String? = null

    @JsonProperty("createdAt")
    var createdAt: Date? = null
}